src/usr/local/www/firewall_nat_nat64.php

```diff
@@ -0,0 +1,290 @@
+<?php
+/*
+ * nat64.inc
+ *
+ * part of pfSense (https://www.pfsense.org)
+ * Copyright (c) 2004-2013 BSD Perimeter
+ * Copyright (c) 2013-2016 Electric Sheep Fencing
+ * Copyright (c) 2014-2020 Rubicon Communications, LLC (Netgate)
+ * Copyright (c) 2020 Nick Whaley
+ * Copyright (c) 2011 Seth Mos <seth.mos@dds.nl>
+ * All rights reserved.
+ *
+ * originally based on m0n0wall (http://m0n0.ch/wall)
+ * Copyright (c) 2003-2004 Manuel Kasper <mk@neon1.net>.
+ * All rights reserved.
```

Comment:
Same comments I made on nat64.inc.  If you copied and changed few, keep it, otherwise, clean it up